docs/using-the-probo-app/projects.md

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
For the most part information in this file can be merged with other pages we already have. The information about Assets and the Probo Uploader can go into [the Assets page](https://docs.probo.ci/build/assets/), and details about Probo Builds can go into [the Build Pages page](https://docs.probo.ci/app/builds/). Otherwise I feel like we are duplicating info, which makes maintaining these docs a pain.

Reply:
With the additional documentation I am adding for this section, I'm trying to step users through the sections available on a specific page in the UI. The current [Builds page](https://docs.probo.ci/app/builds/) is specific to the actual build itself and functions available to the build page. I'm describing what the Builds tab is from the context of the Organization page. They are completely different sections of the UI, and based on the amount of support requests we get about these tabs, there needs to be documentation specific to each page in the UI that matches directly with screenshots of the UI. I don't want to mix it up too much and confuse people grouping everything that has to do with builds onto one page.